### PR TITLE
README.md improvements

### PR DESCRIPTION
**Pull request by LenzGr**
This is a continuation of [PR#429](https://github.com/SUSE/DeepSea/pull/429), incorporating the feedback and further improving the flow and formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,115 +26,317 @@ feedback, please join us there! We look forward to your contribution.
...
+# systemctl disable SuSEfirewall2.service
+```
+
+Install DeepSea on the node which will be the Salt master:
+
+```
+# zypper install deepsea
+```
+
+The command installs the `salt-master` and `salt-minion` packages as a dependency as well as Grafana/Prometheus.
```
Comment (jan--f) [inline README.md]: Grafana and Prometheus are not installed as a package dependency. They are installed by salt in stage 2. I'd suggest simply removing the part here.

File: README.md [hunk 2]
```diff
@@ -18,115 +26,317 @@ feedback, please join us there! We look forward to your contribution.
...
+*Please note that for now the baseline benchmark assumes all uniform OSDs.*
+
+To load test CephFS run:
+
+```
+# salt-run state.orch ceph.benchmarks.cephfs
+```
+
+This requires a running MDS (deploy in stage 4) and at least on minion with the
+`mds-client` role. The `cephfs_benchmark` stage will then mount the CephFS
```
Comment (jan--f) [inline README.md]: The client role name changed recently to ```client-cephfs```. Could you please add this change here?

File: README.md [hunk 3]
```diff
@@ -18,115 +26,317 @@ feedback, please join us there! We look forward to your contribution.
...
+instance on the mds-client and run a bunch of `fio` tests. See the [benchmark
+readme](srv/pillar/ceph/benchmark/README.md) for futher details.
```
Comment (jan--f) [inline README.md]: Also two pre-existing errors here:

- Typo futher -> further
- the readme path changed to ```srv/pillar/ceph/benchmarks/README.md```

File: README.md (hunk 1 above)
Comment (LenzGr) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (LenzGr) [inline README.md]: Thanks for the hint - done.

File: README.md (hunk 3 above)
Comment (LenzGr) [inline README.md]: Good spotting! Fixed.

**Pull request merged**